lib/kredis/types/proxy.rb

```diff
@@ -19,8 +17,8 @@ def method_missing(method, *args, **kwargs)
 
   private
     def log_message(method, *args, **kwargs)
-      args      = args.flatten.compact_blank.presence
-      kwargs    = kwargs.compact_blank.presence
+      args      = args.flatten.reject(&:blank?).presence
+      kwargs    = kwargs.reject(&:blank?).presence
```

Comment:
I'm amazed that our test suite still passes because I think this needs to be `reject { |k, v| v.blank? }` as it's called on a Hash.

Reply:
Honestly, I just cut and pasted the contents of `compact_blank` to make it work. Apparently, hash values can be blank!

Reply:
I am seeing some pretty strange behaviour from your counter type:

```
> counter.increment
Traceback (most recent call last):
        1: from (irb):2
ArgumentError (wrong number of arguments (given 1, expected 0))
```

When I run the test suite, it's telling me: `/home/leastbad/kredis/lib/kredis/types/proxying.rb:7: warning: '*' interpreted as argument prefix`

Reply:
> Honestly, I just cut and pasted the contents of `compact_blank` to make it work. Apparently, hash values can be blank!

Not sure what you mean? There's a distinct `Hash` version: https://github.com/rails/rails/blob/d30dad4bc1039c80648f39c5171b146f8dd8abd6/activesupport/lib/active_support/core_ext/enumerable.rb#L199

Reply:
> I am seeing some pretty strange behaviour from your counter type:

I couldn't reproduce that example with:

```ruby
counter = Kredis.counter "mycounter"
counter.increment
```

Do you have the full backtrace?

> /home/leastbad/kredis/lib/kredis/types/proxying.rb:7: warning: '*' interpreted as argument prefix

Ah I guess `rake` is running the tests with warnings enabled, I disagree with the warning so I'd rather not add the parens 😂

Reply:
You were obviously correct about there being a Hash version, which I'd missed.

Latest commit appears to be working great with Rails 6.0. Thanks for being patient.